docs/docsite/Makefile

```diff
@@ -86,6 +90,15 @@ core_structure: core_generate_rst
 	-ln -sf ../rst/core_index.rst rst/index.rst
 	-ln -sf ../sphinx_conf/core_conf.py rst/conf.py
 
+# Symlink the relevant index into place for building core docs
+gettext_structure: gettext_generate_rst
+	@echo "Creating symlinks in gettext_generate_rst"
+	-ln -sf ../rst/core_index.rst rst/index.rst
+	-ln -sf ../sphinx_conf/core_conf.py rst/conf.py
+
+gettext: gettext_structure
```

Comment:
I haven't absorbed what needs to be done but this doesn't seem quite right. It feels like this command should somehow switch structure depending on whether it was making core or Ansible package docs.

Reply:
Maybe like this:
``` Makefile

gettext: ansible_structure
<TAB> CPUS=$(CPUS) $(MAKE) -f Makefile.sphinx gettext

core_gettext: core_structure   
<TAB> CPUS=$(CPUS) $(MAKE) -f Makefile.sphinx gettext
```